client/src/modules/stock/entry/modals/findPurchase.modal.js

```diff
@@ -79,26 +91,25 @@ function StockFindPurchaseModalController(Instance, Purchase, Notify,
 
   /* ======================= End Grid ======================================== */
 
-  Purchase.search({ detailed: 1, status_id: [2, 4] })
-  .then(function (purchases) {
-    vm.gridOptions.data = purchases;
-  })
-  .catch(Notify.errorHandler);
+  Purchase.search({ detailed : 1, status_id : [2, 4] })
```

Comment:
What is status_id "2" and "4"?  Could you put a comment to let future developers know?

Reply:
@lomamech could you help us with the meaning of each status id ?

Reply:
It looks like they are documented [here](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/models/bhima.sql#L167).  Ideally, we should make some bhConstant types for them, so that we can make a more readable:
```js
Purchase.search({ detailed : 1, status_id : [bhConstant.purchaseTypes.CONFIRMED])
```

This can be done in a separate PR.